rfd/0208-proxy-desktop-session.md

```diff
@@ -97,9 +97,17 @@ message ProxyDesktopSessionResponse {
 
 // Identifies the destination desktop within a specific cluster.
 message TargetDesktop {
+   // Operating system of the target desktop.
+   DesktopOS desktop_os = 1;
```

Comment:
I think it's worth adding the target operating system. The name `ProxyDesktopSession` doesn’t indicate which OS it's for, but the current implementation will only support Windows. Since Linux desktop access is on the roadmap, including the OS in the name should make the API more future-proof.

Reply:
Are we even sure that the behavior at the proxy will be the same for linux desktops - i.e. passing along a bytestream given a name in the form of a string and a cluster name?

Reply:
There's also no real cost in supporting multiple RPCs that do the same thing, so we could just call the rpc `ProxyWindowsDesktopSession` and leave the problem for a later time.

Reply:
This is a good idea, I renamed the RPC to `ProxyWindowsDesktopSession`. Thanks for the suggestion!